CHANGELOG.md

```diff
@@ -2,12 +2,12 @@
 
 
 ## v0.4 (unreleased)
-- Add support form `map`, `slice` and `array` to dig
```

Comment:
Because we don't *explicitly* support only map and slice anymore, but rather all pointer-like and value types that are no primitives